store/postgres/src/deployment_store.rs

```diff
@@ -674,6 +674,24 @@ impl DeploymentStore {
         })
         .await
     }
+
+    /// Runs the SQL `ANALYZE` command in a table.
+    pub(crate) async fn analyze(
+        &self,
+        site: Arc<Site>,
+        entity_type: EntityType,
+    ) -> Result<(), StoreError> {
+        let store = self.clone();
+        self.with_conn(move |conn, _| {
+            let layout = store.layout(conn, site)?;
+            let table = layout.table_for_entity(&entity_type)?;
+            let table_name = &table.qualified_name;
+            let sql = format!("analyze {table_name}");
```

Comment:
It would be nice if `diesel` provided `analyze` in their [pg](https://docs.rs/diesel/1.4.8/diesel/pg/index.html) module, perhaps we can add there someday 🙂